### DISCUSSION TITLE
Erros in v2

### DISCUSSION THREAD

**Discussion by rodolfobolconte** [pyannote/pyannote-audio#859] [Q&A]
I tried to run `pyannote/voice-activity-detection` with version 2 as seen in [Hugging Face](https://huggingface.co/pyannote/voice-activity-detection):
```
from pyannote.audio import Pipeline
pipeline = Pipeline.from_pretrained("pyannote/voice-activity-detection")
output = pipeline("audio.wav")
```
but some errors appear according to each tag for v2 available in the repository.

Installing 2.0.0-wip tag:
```
Failed to build pyannote.audio pyroomacoustics
ERROR: Could not build wheels for pyroomacoustics, which is required to install pyproject.toml-based projects
```

Running with 2.0.a0 and 2.0.a1 tags:
```
from pyannote.audio import Pipeline
ImportError: cannot import name 'Pipeline' from 'pyannote.audio'
```

**Comment by hbredin** [comment]
Looks like you did not follow the [installation instructions](https://github.com/pyannote/pyannote-audio/tree/develop#installation) and this part, in particular:

```bash
pip install https://github.com/pyannote/pyannote-audio/archive/develop.zip
```
